services-js/commissions-app/src/pages/commissions/validationSchema.ts

```diff
@@ -0,0 +1,40 @@
+import * as Yup from 'yup';
```

Comment:
You’ll want to move this probably into a `lib` directory as a sibling of `client` and `server` since both will want to reference it.

Typically avoid putting anything that’s no explicitly a page component into `pages`, since Next.js will try to serve it as a page.